build/configs/rtl8721csm/loadable_apps/defconfig

```diff
@@ -230,10 +230,10 @@ CONFIG_BOARD_ASSERT_SYSTEM_HALT=y
 # SPI Flash driver
 #
 CONFIG_AMEBAD_FLASH_BASE=0x6000
```

Comment:
Please check this address is correct or not. When I test, it should be 0x00.

Reply:
Updated.